### ISSUE TITLE
Tab title with messages to read whitout animation

### ISSUE THREAD

**Issue by emafriedrich** [zhukov/webogram#1917]
I suggest that to use this format

`(1) Telegram`

Is better for mental health than animate title between `Telegram Web` and `1 notification`.

The current notification scheme in the web title is super annoying for me.

Thanks

**Comment by pkrabbit** [comment]
Agreed, the switching text is very distracting and there's no way to disable it.

Also, there is an issue where the tab title keeps flashing even after I've checked the new messages.
